src/angular/components/lock.component.ts

```diff
@@ -50,7 +50,7 @@ export class LockComponent implements OnInit {
         this.pinSet = await this.vaultTimeoutService.isPinLockSet();
         this.pinLock = (this.pinSet[0] && this.vaultTimeoutService.pinProtectedKey != null) || this.pinSet[1];
         this.supportsBiometric = await this.platformUtilsService.supportsBiometric();
-        this.biometricLock = await this.vaultTimeoutService.isBiometricLockSet() && await this.cryptoService.hasKey();
+        this.biometricLock = await this.vaultTimeoutService.isBiometricLockSet() && (await this.cryptoService.hasKey() || this.platformUtilsService.identityClientId !== 'desktop');
```

Comment:
What is the specific reason for the != desktop condition here? This seems like a bit of a hack to use this property. Would it be more appropriate to create some new property that is defined around this condition?

Reply:
We want to only store the keys on desktop since the other `jslib` clients don't provide a secure storage. But yea, I agree that abstracting it away would improve readability and provide a context.

How about `this.platformUtilsService.supportsSecureStorage` or `this.platformUtilsService.canSaveKeys`?

Reply:
I like that, `this.platformUtilsService.supportsSecureStorage`

Reply:
Renamed.